sample/src/main/res/layout/activity_main.xml

```diff
@@ -9,6 +9,25 @@
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
       android:text="@string/hello_world"/>
+  <TextView
+      android:id="@+id/repeated_view_1"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:visibility="gone"
```

Comment:
I like the idea. But I think it should be part of the "Given" of the new test, instead of a new test. As you said, it's to verify the context of this test, not something independent. Agree?